docs/en/stack/ml/df-analytics/ml-feature-importance.asciidoc

```diff
@@ -0,0 +1,35 @@
+[role="xpack"]
+[[ml-feature-importance]]
+=== Understanding results: {feat-imp}
+
+{feat-imp-cap} is calculated for supervised {ml} methods such as 
+<<dfa-regression,{regression}>> and <<dfa-classification,{classification}>>. 
```

Comment:
>  Feature importance is calculated ...

Perhaps this can wait to be augmented in a later PR, but it would be nice if the first sentence provided a very brief explanation of this concept. Something like:

"Feature importance values indicate which fields had the biggest impact on each prediction that is generated by classification or regression analysis.